### PR TITLE
Login storage/ remember me functionality

### PR DESCRIPTION
**Pull request by OrlovG**
…the login page, and the logout button on the home page.

### CODE REVIEW

File: src/components/Login/index.tsx [hunk 1]
```diff
@@ -18,6 +20,26 @@ function LoginForm() {
     }
   };
 
+  const rememberUser = async () => {
+    try {
+      await AsyncStorage.setItem('loggedIn', true.toString());
```
Comment (RobinJehn) [inline src/components/Login/index.tsx:26]: Instead of storing True/False, you should store the specific credentials.

File: .erb/configs/webpack.config.base.ts [hunk 2]
```diff
@@ -50,11 +50,14 @@ const configuration: webpack.Configuration = {
     fallback: {
       // crypto: require.resolve('crypto-browserify'),
       // stream: require.resolve('stream-browserify'),
-      // timers: require.resolve('timers-browserify'),
+      // // timers: require.resolve('timers-browserify'),
```
Comment (RobinJehn) [inline .erb/configs/webpack.config.base.ts]: I think all changes in this file can be reverted

File: src/components/RegistrationForm/index.tsx [hunk 3]
```diff
@@ -141,7 +141,7 @@ function RegistrationForm() {
         </button>
       </div>
       <div>
-        <NavLink to="/login" className="has-account-link">
+        <NavLink className="has-account-link" to="/login">
```
Comment (RobinJehn) [inline src/components/RegistrationForm/index.tsx]: why this change?

File: src/renderer/App.tsx [hunk 4]
```diff
@@ -7,28 +12,60 @@ import Welcome from 'components/WelcomePage';
...
-        <Route path="/" element={<Welcome />} />
-        <Route path="/login" element={<LoginForm />} />
-        <Route path="/register" element={<RegistrationForm />} />
-        <Route path="/settings" element={<SettingsPage />} />
-        <Route path="/home" element={<Home />} />
-        <Route path="/seat" element={<Seat />} />
-        <Route path="/eye" element={<EyeTrackingPage />} />
-      </Routes>
-    </Router>
+    <div className="home_page">
```
Comment (RobinJehn) [inline src/renderer/App.tsx]: Why the need for the div

File: src/renderer/App.tsx [hunk 5]
```diff
@@ -7,28 +12,60 @@ import Welcome from 'components/WelcomePage';
...
+      <Router>
+        <Routes>
+          <Route
+            path="/"
+            element={
+              <PrivateRoute>
```
Comment (RobinJehn) [inline src/renderer/App.tsx:19]: Can we use "useNavigate" instead of the private route stuff?

**Comment by RobinJehn** [comment]
Added a couple comments, please address them.

File: src/components/HomePage/index.tsx [hunk 6]
```diff
@@ -28,6 +39,16 @@ function Home() {
           at work. Embrace good office health with HealthSpace Solutions.
         </p>
       </div>
+      <form id="logout_form">
```
Comment (RobinJehn) [inline src/components/HomePage/index.tsx]: I think the logout button should rather be in settings

File: src/components/Login/index.tsx (hunk 1 above)
Comment (OrlovG) [reply src/components/Login/index.tsx:26]: It is not doing any authentication or anything like that. It stores the value true/false, purely just to check if the user wanted to be remembered or not. So really storing credentials, wouldn't change anything at all.

File: src/renderer/App.tsx (hunk 5 above)
Comment (OrlovG) [reply src/renderer/App.tsx:19]: useNavigate or anything else doesn't seem to produce anything that works properly. PrivateRoute seems to be the only method which works, without giving blank pages or stopping after restart.

File: src/components/Login/index.tsx (hunk 1 above)
Comment (RobinJehn) [reply src/components/Login/index.tsx:26]: We would want it either way, like what would happen if the account was deleted on another machine? We should automatically be locked out on this machine too.

File: src/components/HomePage/index.tsx (hunk 6 above)
Comment (RobinJehn) [reply src/components/HomePage/index.tsx]: @OrlovG Please add the logout button again, but to the settings page. Don't remove it completely

File: src/components/Login/index.tsx (hunk 1 above)
Comment (RobinJehn) [reply src/components/Login/index.tsx:26]: Lets ignore that for now, probably don't actually need this for this course.

**Review by RobinJehn — APPROVED** [review]
lgtm
